### ISSUE TITLE
PropTypes error when using code from docs

### ISSUE THREAD

**Issue by tristan957** [Semantic-Org/Semantic-UI-React#3706]
### Steps
The `error` prop on `Form.Input` is typed as `boolean | undefined`, yet this code example, https://react.semantic-ui.com/collections/form/#states-field-error-label, uses an object to display an error.

### Expected Result
Use an object to add an error to a field

### Actual Result
```
Warning: Failed prop type: Invalid prop `error` of type `object` supplied to `FormField`, expected `boolean`.
    in FormField (created by FormInput)
    in FormInput (at ContactPage.tsx:40)
    in form (created by Form)
    in Form (at ContactPage.tsx:32)
    in div (created by Container)
    in Container (at ContactPage.tsx:26)
    in ContactPage (created by Context.Consumer)
    in Route (at App.tsx:43)
    in Switch (at App.tsx:37)
    in div (at App.tsx:34)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App.tsx:33)
    in App (at src/index.tsx:9)
```

### Version
0.87.1

**Comment by layershifter** [comment]
This functionality was added in #3603 and was released in `0.87.3`:
https://codesandbox.io/s/jbil6?module=/example.js

**Comment by tristan957** [comment]
Thank you
